compiler/damlc/daml-stdlib-src/DA/Internal/Interface/AnyView.daml

```diff
@@ -0,0 +1,19 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module DA.Internal.Interface.AnyView (InterfaceTypeRep (..), AnyView (..)) where
```

Comment:
I like the approach of other `StablePackages` where the stable module is named `Foo.Bar.Types` and the non-stable just `Foo.Bar`

Reply:
done